frontends/p4-14/p4-14-parse.ypp

```diff
@@ -66,6 +66,8 @@ static const IR::Annotations *getPragmas() {
     IR::Annotations             *Annotations;
     IR::Apply                   *Apply;
     IR::Attribute               *Attribute;
+    IR::AttribLocal             *AttribLocal;
```

Comment:
I really wish you guys would migrate to P4-16 and stopped extending P4-14.
I think that the simplest way to do attributes in P4-16 is to model them as constructor parameters, and to allow a don't care initializer for a constructor parameter. Hopefully you don't really need a string type...